.gitignore

```diff
@@ -11,3 +11,4 @@
 .rspec_status
 
 /Gemfile.lock
+/gemfiles/*.lock
```

Comment:
might be better to do something like `*emfile.lock` but I'm not really picky about that! Thanks for the work

Reply:
I can replace with this. I would remove then the initial slash:
`*emfile.lock` instead of L13-14?